src/components/AddFacilityModal/AddFacilityModal.jsx

```diff
@@ -8,11 +8,12 @@ import {
   Button,
   useDisclosure,
   Input,
-  Avatar,
+  // Avatar,
 } from '@chakra-ui/react';
 // import { AddIcon } from '@chakra-ui/icons';
 import React from 'react';
 import './AddFacilityModal.css';
+import UploadAvatar from '../UploadAvatar/UploadAvatar';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'UploadAvatar' is defined but never used.